python/cudf/cudf/_lib/pylibcudf/interop.pyx

```diff
@@ -170,6 +208,18 @@ def to_arrow(cudf_object, metadata=None):
     raise TypeError("to_arrow only accepts Table and Scalar objects")
 
 
+@to_arrow.register(DataType)
+def _to_arrow_datatype(cudf_object):
+    if cudf_object.id() == type_id.DECIMAL128:
+        return pa.decimal128(-cudf_object.scale)
```

Comment:
Looks like the signature here is `precision: int, scale: int = 0` so shouldn't scale be the second argument here?